content/sensu-go/5.5/reference/agent.md

```diff
@@ -247,9 +286,17 @@ required     | true
 type         | String
 example      | {{< highlight shell >}}"output": "CheckHttp OK: 200, 78572 bytes"{{< /highlight >}}
 
+source       | 
+-------------|------
+description  | The name of the Sensu entity associated with the event. Use this attribute to tie the event to a proxy entity. If no matching entity exists, Sensu creates a proxy entity with the name provided by the `source` attribute.
+required     | false
+default      | The agent entity receiving the event data
+type         | String
+example      | {{< highlight shell >}}"source": "sensu-docs-site"{{< /highlight >}}
+
 client       | 
 -------------|------
-description  | The name of the Sensu entity associated with the event. The `client` attribute gives you the ability to tie the event to a proxy entity while providing compatibility with [Sensu 1.x check results][42]. Use this attribute to specify the name of the [proxy entity][43] tied to the event.
+description  | The name of the Sensu entity associated with the event. Use this attribute to tie the event to an existing proxy entity.
```

Comment:
The `client` field still behaves the same. It also behaves the same as the new `source` field. It's just not recommended for use anymore.